relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -1,15 +1,218 @@
 use std::collections::BTreeMap;
+use std::fmt::{self, Display};
 
-use relay_common::{UnixTimestamp, Uuid};
+use relay_common::{MetricUnit, UnixTimestamp, Uuid};
 use relay_general::protocol::{
     AbnormalMechanism, SessionAttributes, SessionErrored, SessionLike, SessionStatus,
 };
-use relay_metrics::{Metric, MetricNamespace, MetricUnit, MetricValue};
+use relay_metrics::{
+    CounterType, Metric, MetricNamespace, MetricResourceIdentifier, MetricType, MetricValue,
+};
+
+/// Enumerates the most common session-names.
+#[derive(Clone, Debug, PartialEq)]
+pub enum SessionsKind {
```

Comment:
This is what it represents.

```suggestion
pub enum SessionMetric {
```